.gitignore

```diff
@@ -92,3 +92,4 @@ Thumbs.db
 UserSettings/Layouts/default-2021.dwlt
 
 VersionControlSettings.asset
+ProjectSettings
```

Comment:
This is a valid directory and should NOT ignore.

I suggest to get the latest version from https://github.com/github/gitignore/blob/main/Unity.gitignore.